src/ngraph/op/softmax.cpp

```diff
@@ -90,3 +97,70 @@ void op::Softmax::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVect
     auto x = input_value(0);
     adjoints.add_delta(x, adjoint);
 }
+
+// *** SOFTMAX OP SET V1 ***
+const string op::v1::Softmax::type_name{"Softmax"};
+
+op::v1::Softmax::Softmax(const Output<Node>& arg, const size_t axis)
+    : Op({arg})
+    , m_axis(axis)
+{
+    constructor_validate_and_infer_types();
+
+    const PartialShape& input_shape = get_input_partial_shape(0);
+    NODE_VALIDATION_CHECK(this,
+                          axis >= 0 && axis < static_cast<size_t>(input_shape.rank()),
+                          "Reduction axis (",
+                          axis,
+                          ") is out of bounds (argument shape: ",
+                          input_shape,
+                          ").");
+
+    if (input_shape.is_static())
+        set_output_type(0, get_input_element_type(0), input_shape.to_shape());
+    else
+        set_output_type(0, get_input_element_type(0), PartialShape::dynamic());
+}
+
+shared_ptr<Node> op::v1::Softmax::copy_with_new_args(const NodeVector& new_args) const
+{
+    check_new_args_count(this, new_args);
+    return make_shared<op::v1::Softmax>(new_args.at(0), m_axis);
+}
+
+void op::v1::Softmax::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVector& deltas)
+{
+    throw ngraph_error("NYI");
```

Comment:
I am opposed to any abbreviations. Please remove this New York Institute and replace with a meaningful sentence, such as: `op::v1::Softmax::generate_adjoints function is not implemented yet.`

Reply:
Done